lib/pluginmanager/sources/local.rb

```diff
@@ -0,0 +1,26 @@
+# encoding: utf-8
+require "rubygems/package"
+require "pluginmanager/sources/base"
+require "fileutils"
+
+module LogStash::PluginManager::Sources
+
+  class Local < Base
+
+    def exist?
+      ::File.exist?("#{uri}")
+    end
+
+    def fetch(dest="")
+      base_dir = (dest.empty? ? LogStash::Environment::LOGSTASH_HOME : dest )
+      current_path = ::File.join(base_dir, ::File.basename("#{uri}"))
+      ::FileUtils.cp("#{uri}", current_path)
+      [ current_path, "200" ]
+    end
```

Comment:
"200" is an implementation details which doesn't have any relation with the local file system.
Would it better to return an object with a success method?